conftest.py

```diff
@@ -136,6 +136,81 @@ def test_table(request, sql_server):  # noqa: C901
             hook.run(f"DROP SCHEMA IF EXISTS {table.metadata.schema} CASCADE;")
 
 
+@pytest.fixture
+def table_fixture(request, sql_server):  # noqa: C901
+    tables = []
+    tables_params = [{}]
+
+    if getattr(request, "param", None):
+        if isinstance(request.param, list):
+            tables_params = request.param
+        else:
+            tables_params = [request.param]
+
+    sql_name, hook = sql_server
+    database = get_database_name(hook)
+
+    for table_param in tables_params:
+        load_table = table_param.get("load_table", False)
+        override_table_options = table_param.get("param", {})
+
+        if database == Database.SNOWFLAKE:
+            default_table_options = {
+                "conn_id": hook.snowflake_conn_id,
+                "metadata": Metadata(
+                    database=hook.database,
+                    warehouse=hook.warehouse,
+                    schema=hook.schema,
+                ),
+            }
+        elif database == Database.POSTGRES:
+            default_table_options = {
+                "conn_id": hook.postgres_conn_id,
+                "metadata": Metadata(database=hook.schema),
+            }
+        elif database == Database.SQLITE:
+            default_table_options = {
+                "conn_id": hook.sqlite_conn_id,
+                "metadata": Metadata(
+                    database="sqlite",
+                ),
+            }
+        elif database == Database.BIGQUERY:
+            default_table_options = {
+                "conn_id": hook.gcp_conn_id,
+                "metadata": Metadata(
+                    schema=SCHEMA,
+                ),
+            }
```

Comment:
I strongly believe we should be moving out of this old pattern and using the `Database` class, similar to the `database_table_fixture`

Reply:
This was reintroduced in rebase. Removed it.